### ISSUE TITLE
Tuning parameters

### ISSUE THREAD

**Issue by syuntoku14** [syuntoku14/flow_rl_project#21]
|         |                                 algorithm                                 |       rew       |         obs         |                                    eta2                                   |                             t_min                             | randomness                                                                      |
|:-------:|:-------------------------------------------------------------------------:|:---------------:|:-------------------:|:-------------------------------------------------------------------------:|:-------------------------------------------------------------:|---------------------------------------------------------------------------------|
|         |                                    PPO                                    |     default     |       default       |                            default(0.1 (good))                            |                    default(1.0(too small))                    | default(Flow rate, merge rate, etc are constant)                                |
|         |                                    DDPG                                   |  + outflow rate | + distance to merge |                              0.5(a bit large)                             |                          10.0(good?)                          | cars depart randomly                                                            |
|         |                                                                           | historical data |   10.0(too large)   |                                                                           |                                                               |                                                                                 |
| remarks | PPO is better? Reward design and hyperparameter tuning are more important |        NA       |          NA         | If too large, car stops. If the small, car will slow down but reach the rear. | Large t_min reduce the congestion but reduce the outflow too. | The paper only considers the constant flow rate case, and it may overfit the problem |
